packages/react/src/components/Button/BaseButton.tsx

```diff
@@ -629,7 +629,7 @@ export class BaseButton extends React.Component<IBaseButtonProps, IBaseButtonSta
         ref={this._splitButtonContainer}
         data-is-focusable={true}
         onClick={!disabled && !primaryDisabled ? this._onSplitButtonPrimaryClick : undefined}
-        tabIndex={!disabled || allowDisabledFocus ? 0 : undefined}
+        tabIndex={(!disabled || allowDisabledFocus) && !primaryDisabled ? 0 : undefined}
```

Comment:
I think this should be 

```suggestion
        tabIndex={(!disabled && !primaryDisabled) || allowDisabledFocus ? 0 : undefined}
```

Reply:
Makes sense - will make the change!